_posts/2018-09-08-terminal-tips.md

```diff
@@ -4,3 +4,26 @@ tags: linux, bash, compilation
 ---
 
 // Start your contributions here
+*Contributed by: <your name>*
+### Title for the tip/code
+    insert code here (if any)
+description about what your tip does with accompanying code snippet
+
+---
+
+*Contributed by: Bhargav SNV*
+### Stop/Start Displaying Terminal Input
```

Comment:
I guess it would be better phrased as 
When `stty -echo` is *run* instead of *done*

Reply:
My bad, will change